sdk/keyvault/azure-keyvault-certificates/samples/backup_restore_operations.py

```diff
@@ -55,12 +55,12 @@
     # Backups are good to have, if in case certificates gets deleted accidentally.
     # For long term storage, it is ideal to write the backup to a file.
     print("\n.. Create a backup for an existing certificate")
-    certificate_backup = client.backup_certificate(name=cert_name)
+    certificate_backup = client.backup_certificate(certificate_name=cert_name)
```

Comment:
When there's only one positional parameter, it's reasonable not to pass it by name.